shared/gh/api/gh.api.util.js

```diff
@@ -484,14 +486,21 @@ define(['exports', 'moment', 'sinon', 'bootstrap-notify'], function(exports, mom
             'declarePartial': function(name, template) {
                 partialCache[name] = _.template(template);
             },
-            'partial': function(name, data) {
+            'partial': function(name, data, renderAtStart) {
+                // TODO: replace `renderStart` with a more robust solution for delayed rendering
+                //       of partials inside of partials
+                if (renderAtStart === false) {
+                    return '<%= _.partial("' + name + '", {data: data}, null) %>';
+                }
```

Comment:
I'll just ignore the if for now, we should come up with a better approach to this.